Documentation/Internal/Grafana/ama-metrics-prometheus-config.yaml

```diff
@@ -0,0 +1,71 @@
+kind: ConfigMap
+apiVersion: v1
+data:
+  prometheus-config: |-
+    scrape_configs:
+    - job_name: ama-logs-daemonset-linux
+      kubernetes_sd_configs:
+        - role: pod
+      relabel_configs:
+        - source_labels: [__meta_kubernetes_pod_controller_kind]
```

Comment:
do you know why some labels is in this format __<lable>__ vs __<lable>

Reply:
> do you know why some labels is in this format vs __

These are standard meta labels - https://prometheus.io/docs/prometheus/latest/configuration/configuration/#pod